test-harness/modules/mocker/endpoints/mesos.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+"""Mesos mock endpoint"""
+
+import logging
+
+from exceptions import EndpointException
+from mocker.endpoints.recording import (
+    RecordingHTTPRequestHandler,
+    RecordingTcpIpEndpoint,
+)
+
+# pylint: disable=C0103
+log = logging.getLogger(__name__)
+
+
+# pylint: disable=R0903
+class MesosHTTPRequestHandler(RecordingHTTPRequestHandler):
+    """A request hander class mimicking DC/OS IAM (Bouncer) operation.
```

Comment:
> ????

The general thought behind my recommendation was that sometimes it's better to not have additional information than have information that is either outdated, misleading, or nothing new (duplicate).

So, I didn't mean to say that I expect perfect docstrings, and I didn't mean to question the usefulness of _awesome_ docstrings. But doing an _awesome_ docstring is a lot of work and requires some love, and if we don't have the time, then it's sometimes better to not add a docstring. Hence my harsh statement "remove them if it's too much of a burden to maintain them" -- I've found this to be an important trade-off when operating under time pressure.

I'm sure you're making the right call here, my feedback is really meant just in a general way.